### DISCUSSION TITLE
Unrecognised character for markdown files under an H* element

### DISCUSSION THREAD

**Discussion by sukantamaikap** [LazyVim/LazyVim#2377] [Q&A] [answered]
I'm noticing these unrecognised characters **only** in `markdown` files under `H*` elements like below:

<img width="979" alt="Screenshot 2024-01-18 at 9 24 55 pm" src="https://github.com/LazyVim/LazyVim/assets/3388408/f88320fd-372f-46e2-96b1-ad4fa3eed105">

I'm using Iterm2 with Nerd Font on Mac (OSX 14.2.1). Please note that these characters only appear for markdown files only!! I do not see the same if I open the same file using vim. Please do help.

**Comment by dpetka2001** [comment, marked as answer]
Unfortunately, it's a problem with how your fonts are rendering. See this [comment](https://github.com/lukas-reineke/headlines.nvim/issues/41#issuecomment-1556334775) in the official plugin repo.

**Reply by sukantamaikap** [reply]
That worked!! Thanks a lot.
